src/commands/lambda/prompt.ts

```diff
@@ -305,3 +306,21 @@ export const requestFunctionSelection = async (functionNames: string[]) => {
     }
   }
 }
+
+export const requestFilePath = async () => {
```

Comment:
No unit tests for this method in `prompt.test.ts`

Reply:
Nice catch. Unit tests added